buildpack.toml

```diff
@@ -0,0 +1,14 @@
+api = "0.7"
+
+[buildpack]
+  id = "ninech/buildpack-static-confgen"
+  homepage = "https://github.com/ninech/buildpack-static-confgen"
+  name = "Static Website Buildpack"
```

Comment:
I noticed that this is the same name as used in the `static-require` buildpack. Can we maybe differentiate them?

Reply:
Good point, renamed it to `Static Website Config Generator`